saleor/giftcard/models.py

```diff
@@ -0,0 +1,25 @@
+from datetime import date
+
+from django.conf import settings
+from django.db import models
+from django_prices.models import MoneyField
+
+
+class GiftCard(models.Model):
+    code = models.CharField(max_length=16, unique=True, db_index=True)
```

Comment:
Maybe it could be set by a settings value?

Reply:
For vouchers, we don't set code length using settings. I think this feature can be separate PR.